tracing-opentelemetry/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'io.micronaut.build.internal.tracing-module'
+}
+
+dependencies {
+    api mn.micronaut.runtime
+    api mn.micronaut.http.client
+    api mn.micronaut.grpc.server.runtime
```

Comment:
I think we better split it into modules or add all initializers into one module but activate only if the class is provided in the classpath. For example to activate gRPC interceptors you should add:

```
implementation("io.opentelemetry.instrumentation:opentelemetry-grpc-1.6")
```

Reply:
Same for Micronaut HTTP client as well.